flocker/acceptance/test_api.py

```diff
@@ -89,46 +90,25 @@ def test_create_container_with_environment(self, cluster):
         Create a container including environment variables on a single-node
```

Comment:
This docstring has made the review of this issue harder - what I should be looking at mostly is whether the new test fits the description. Instead I am grokking the old test and looking at the new test to make sure that it is missing nothing I guess is important. I'm working on the assumption that it should be more like:

"If environment variables are specified when creating a container, those variables are available in the container's environment".